### ISSUE TITLE
Unreadable sign during multiplayer resource pack check.

### ISSUE THREAD

**Issue by davepusey** [NeunEinser/bingo#241]
Tested in 5.1.4-beta1 on 1.21.2-rc1

The "Resource pack was installed. Click here for chat message." sign is completely unreadable.

![original](https://github.com/user-attachments/assets/fa3f397c-24d5-4529-9346-f064fb2d08f5)

Seems to be due to a lack of lighting, as placing a light source directly below the player fixes it...

/setblock 1 0 1 minecraft:sea_lantern

![with added light source](https://github.com/user-attachments/assets/d503cab5-6cfa-459b-963d-cb6c90f66ac3)

**Comment by NeunEinser** [comment]
This is a lighting glitch, the sea lantern behind the sihn also lights the room. If you remove the one below again, the light should stay.

Pretty sure this is not really new and happened before like this already, although I don't think it is consistent and shouldn't happen everytime.

**Comment by davepusey** [comment]
Yeah I think this was an issue in 5.1.3 too, as my customised version for Phedran's streams has the extra sea lantern on the floor too.
